README.markdown

```diff
@@ -187,49 +233,76 @@ If Puppet is running on a 64-bit system, the 32-bit Registry key can be explicit
 
 For more on this parameter, see the [Purge existing values section](#purge-existing-values) under Usage.
 
+
 #### `registry_value`
 
-##### `path`
+Manages individual Registry values.
 
-*Optional.* Specifies a Registry value for Puppet to manage. Valid options: a string containing a Registry path. If any of the parent keys in the path don't exist, Puppet creates them automatically. For example: `HKLM\Software` or `HKEY_LOCAL_MACHINE\Software\Vendor`. Default value: the title of your declared resource.
+The title of the resource can determine the path and value name that will be managed, for example:
 
-If Puppet is running on a 64-bit system, the 32-bit Registry key can be explicitly managed using a prefix. For example: `32:HKLM\Software\Value3`.
+The manifest below would manage the value called 'foo' in the key 'HKLM\Software'
 
-##### `ensure`
+``` puppet
+registry_value { 'HKLM\Software\foo':
+  ...
+}
+```
 
-Tells Puppet whether the value should or shouldn't exist. Valid options: 'present' and 'absent'. Default value: 'present'.
+You can also manage the default value of a key by adding a backslash at the end of the path.  The manifest below would manage the default value in the key 'HKLM\Software\Vendor'
```

Comment:
Suggestion: omit would. "The manifest below manages the..."

Reply:
Fixed